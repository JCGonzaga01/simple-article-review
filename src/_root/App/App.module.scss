.App {
  @media (max-width: 845px) {
    margin: 39px 20px 50px;
  }
  
  margin: 39px 93px 50px;
}
