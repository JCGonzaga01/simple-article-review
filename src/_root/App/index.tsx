import React from "react";
import Header from "_root/Header";
import Footer from "_root/Footer";
import styles from "./App.module.scss";

const App: React.FC = (props) => {
  return (
    <div className={styles.App}>
      <Header />
      <div>{props.children}</div>
      <Footer />
    </div>
  );
};

export default App;
