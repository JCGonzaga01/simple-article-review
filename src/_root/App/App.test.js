import React from "react";
import renderer from "react-test-renderer";
import App from "./index";

describe("<App />", () => {
  test("It should mount", () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
