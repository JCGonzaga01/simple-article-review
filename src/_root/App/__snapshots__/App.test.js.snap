// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> It should mount 1`] = `
<div
  className="App"
>
  <div
    className="wrapper"
  >
    <img
      alt="adrenalin"
      className="adrenalin"
      src="/assets/adrenalin.svg"
    />
    <div
      className="headerLinks"
    >
      <span>
        Culture
      </span>
      <span>
        Work
      </span>
      <span>
        Clients
      </span>
      <span>
        Services
      </span>
      <span>
        Careers
      </span>
      <span>
        Contact
      </span>
    </div>
  </div>
  <div />
  <div
    className="wrapper"
  >
    <img
      alt="adrenalin"
      className="adrenalin"
      src="/assets/adrenalin.svg"
    />
    <div
      className="footerLinks"
    >
      <span>
        Privacy
      </span>
      <span>
        Sitemap
      </span>
      <span>
        Facebook
      </span>
      <span>
        LinkedIn
      </span>
      <span>
        Instagram
      </span>
      <span>
        Twitter
      </span>
    </div>
  </div>
</div>
`;
